src/test/java/org/openlmis/requisition/service/RequisitionServiceTest.java

```diff
@@ -1037,6 +1037,43 @@ public void shouldFindAndSortAndPageApprovedRequisitions() {
   }
 
   @Test
+  public void shouldFilterRequisitionsForConvertByMultipleValues() {
```

Comment:
we should add a test for facility search as well. "all' would be good to test as the param as well